DESCRIPTION

```diff
@@ -1,5 +1,5 @@
 Package: protti
-Title: Proteomics data analysis from the Picotti lab
+Title: Proteomics Data Analysis from the Picotti Lab
```

Comment:
R Package for Proteomics Quality Control and Data Analysis developed in the Picotti Lab.